worker/src/RTC/TransportCongestionControlClient.cpp

```diff
@@ -163,9 +164,57 @@ namespace RTC
 	{
 		MS_TRACE();
 
+		// Update packet loss history
+		size_t expected_packets = feedback->GetPacketStatusCount();
+		size_t lost_packets = 0;
+		for (const auto& result : feedback->GetPacketResults())
+		{
+			if (!result.received)
+				lost_packets += 1;
+		}
+		this->UpdatePacketLoss(static_cast<double>(lost_packets) / expected_packets);
+
 		this->rtpTransportControllerSend->OnTransportFeedback(*feedback);
 	}
 
+	void TransportCongestionControlClient::UpdatePacketLoss(double packetLoss)
+	{
+		// Add the score into the histogram.
+		if (this->packetLossHistory.size() == PacketLossHistogramLength)
+			this->packetLossHistory.erase(this->packetLossHistory.begin());
```

Comment:
I feel like this will start moving things around on every call once the history grows to `packetLossHistory` size, maybe `std::deque` is a better data structure here?

Reply:
or `std::queue`

Reply:
I used the same approach used for the scores history in RtpStream.hpp for consistency.
```std::vector<uint8_t> scores;```

I will change it in this PR to use `std::queue`